lib/carrierwave_data_uri/tempfile.rb

```diff
@@ -6,7 +6,7 @@ class Tempfile < ::Tempfile
       attr_writer :original_filename
 
       def original_filename
-        @original_filename || File.basename(@tmpname)
+        @original_filename
```

Comment:
Sorry. It's my mistake. so, `@original_filename || File.basename(@tmpname)` works well.

I delete the code when investigation below.

(this is not related to this mistake)
When strong parameter has both attr `:image, :image_data_uri`.
`version` method not works well (like `version :thumb do`).